test/specs/annotation.tests.ts

```diff
@@ -0,0 +1,66 @@
+import * as chai from 'chai';
```

Comment:
Can you move this to anywhere in the `expressions` folder? Over there it will only be ran in the normal unit tests